cms/static/cms/js/require-config.js

```diff
@@ -1,289 +1,315 @@
-if (window) {
-    // MathJax Fast Preview was introduced in 2.5. However, it
-    // causes undesirable flashing/font size changes when
-    // MathJax is used for interactive preview (equation editor).
-    // Setting processSectionDelay to 0 (see below) fully eliminates
-    // fast preview, but to reduce confusion, we are also setting
-    // the option as displayed in the context menu to false.
-    // When upgrading to 2.6, check if this variable name changed.
-    window.MathJax = {
-      menuSettings: {CHTMLpreview: false}
-    };
-}
+;(function (require, define) {
+    'use strict';
+    if (window) {
+        // MathJax Fast Preview was introduced in 2.5. However, it
+        // causes undesirable flashing/font size changes when
+        // MathJax is used for interactive preview (equation editor).
+        // Setting processSectionDelay to 0 (see below) fully eliminates
+        // fast preview, but to reduce confusion, we are also setting
+        // the option as displayed in the context menu to false.
+        // When upgrading to 2.6, check if this variable name changed.
+        window.MathJax = {
+            menuSettings: {CHTMLpreview: false}
+        };
+        // We do not wish to bundle common libraries (that may also be used by non-RequireJS code on the page
+        // into the optimized files. Therefore load these libraries through script tags and explicitly define them.
+        // Note that when the optimizer executes this code, window will not be defined.
+        var defineDependency = function (globalVariable, name, noShim) {
```

Comment:
OK, let's just simplify this whole block of code (lines 14-30) to 

define("gettext", [] function () { return window["gettext"]; });

(with an appropriate comment of course)

I still think you want to delete the "/i18n" path definition below, and also in the LMS requireJS config.